packages/basic-auth/lib/basic-auth-construct.ts

```diff
@@ -19,6 +19,8 @@ export class BasicAuthFunction extends Construct {
             this,
             'BasicAuthFunction',
             {
+              runtime: Runtime.NODEJS_12_X,
```

Comment:
We should probably update all of these runtimes to node 16 like our serverless projects. Did we want to do that as part of this upgrade or we can create a separate ticket?

Reply:
Yeah I noticed this too! I was going to add it to our audit sheet, thanks for the reminder :smile:. I'll make the changes in this repo as part of another ticket. (I figured upgrading everything to CDK 2 was already a huge change so I didn't want to throw in a node runtime change too)